pipeline/pipeline_tasks/api_calls/geocoder.py

```diff
@@ -1 +1,56 @@
-"""Hit Google Maps API to geocode data set"""
+"""
+Geocoding Utility:
+"""
+import os
+
+import pandas as pd
+import sqlalchemy as sqla
+from sqlalchemy import create_engine
+
+from geopy import GoogleV3
+from utilities.db_manager import DBManager
+
+SBA_DWH = os.getenv('SBA_DWH')
+engine = create_engine(SBA_DWH)
+GOOGLE_MAPS_API_KEY = os.getenv('GOOGLE_MAPS_API')
+
+
+def geocode_address(address, api_key=GOOGLE_MAPS_API_KEY):
+    """Geocodes a single address
+
+    Keyword Args:
+    address: string with the address"""
+    geolocator = GoogleV3(api_key=api_key)
+    query_result = geolocator.geocode(address)
+    if query_result != None:
+        latitude = query_result.latitude
+        longitude = query_result.longitude
+    else:
+        latitude = "Not found"
```

Comment:
Can we use None instead of "Not found" here, that way the column will be stored as a double instead of text in the DB.